spec/requests/healthcheck_spec.rb

```diff
@@ -5,7 +5,7 @@
     get "/healthcheck"
 
     expect(response.status).to eq(200)
-    expect(response.content_type).to eq("application/json")
+    expect(response.content_type).to eq("application/json; charset=utf-8")
```

Comment:
This behaviour is controlled by the: ```Rails.application.config.action_dispatch.return_only_media_type_on_content_type = false``` setting. I've assumed the default setting (including the charset) is preferable; if not, it would be easy to exclude it.